server/src/main/kotlin/org/javacs/kt/signaturehelp/SignatureHelp.kt

```diff
@@ -23,7 +23,7 @@ import org.javacs.kt.LOG
 
 fun fetchSignatureHelpAt(file: CompiledFile, cursor: Int): SignatureHelp? {
     val (signatures, activeDeclaration, activeParameter) = getSignatureTriplet(file, cursor) ?: return nullResult("No call around ${file.describePosition(cursor)}")
-    return SignatureHelp(signatures, activeDeclaration, activeParameter)
+    return SignatureHelp(signatures, (if (activeDeclaration >= 0) activeDeclaration else null), (if (activeDeclaration >= 0) activeParameter else null))
```

Comment:
I think it would be better to log a warning or error if this is a condition that shouldn't occur, to avoid potentially masking underlying issues.

Reply:
Yeah, the Detekt rules can be a bit annoying. For now, I'd just put a

```kotlin
@Suppress("ReturnCount")
```

on those methods.